Tests/Templating/Fixture/FooBundle/Action/FooAction.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Sensio\Bundle\FrameworkExtraBundle\Tests\Templating\Fixture\FooBundle\Action;
+
+class FooAction
+{
+}
```

Comment:
Can you add some test cases to verify `@Route` and `@Template` are still working on this for different cases? `__invoke()`, `someAction()`, `fooBar`() etc. The last one is based on @dunglas suggestion regarding the Action suffix.